src/content/en/updates/2016/10/touch-action.md

```diff
@@ -0,0 +1,95 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Touch actions allow a developer to define how a user can interact
+with an element and Chrome now has wider support for touch-action options in
+Chrome 55.
+
+{# wf_published_on: 2016-10-20 #}
+{# wf_updated_on: 2016-10-20 #}
+{# wf_featured_image: /web/updates/images/generic/focus.png #}
+{# wf_tags: chrome55,input #}
+{# wf_featured_snippet: Touch action is a simple way to define how a user can
+  interact with an element. #}
+
```

Comment:
In updates it's the featured snippet that's seen, not the description. Is this what you really want? I ask because you've used different text for them.